lessons/6_objects_intro/index.html

```diff
@@ -294,7 +294,7 @@ <h3>Class work</h3>
         </section>
         <section>
             <h3>Homework</h3>
-            <a href="https://mrtim.github.io/js-munich-2020-spring/lessons/6_objects_intro/homework">
+            <a href="https://mrtim.github.io/js-munich-2020-spring/homework/5_store">
```

Comment:
Should not it be the 6th homework? There was a homework from lesson 5.

Reply:
ahh good call. I moved the 5th homework markdown into this structure as well